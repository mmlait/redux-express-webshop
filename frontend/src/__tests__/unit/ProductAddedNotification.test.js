import React from 'react';
import { create } from 'react-test-renderer'
import ProductAddedNotification from '../../components/ProductAddedNotification';

describe('ProductAddedNotification component', () => {
    it('should render', () => {
    let tree = create(<ProductAddedNotification />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})