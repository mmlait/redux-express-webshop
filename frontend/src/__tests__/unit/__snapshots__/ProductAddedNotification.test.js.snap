// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductAddedNotification component should render 1`] = `
<div
  className="sc-bdVaJa gtzOcq"
  id="productAddedNotification"
>
  <p>
    A new product was added!
  </p>
</div>
`;
